src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -133,6 +133,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->scalarNode('error_controller')
                     ->defaultValue('error_controller')
                 ->end()
+                ->booleanNode('catch_throwable_exception')->defaultFalse()->info('Tells HttpKernel to catch every kind of Throwable')->end()
```

Comment:
Maybe we could/should rename `catch_throwable_exception` to something like `catch_all_throwables` as not all throwables are exceptions.

Reply:
Yeah. I was hesitant of this name too. I like your suggestion